### ISSUE TITLE
Exercise idea for Reusable

### ISSUE THREAD

**Issue by selgebali** [LibraryCarpentry/lc-fair-research#70]
The idea below is taken from the Sprint [GoogleDoc](https://docs.google.com/document/d/124pu7Ufy3QKzGhowId2PtwvQ5LNJU35UpZ9qxbl1ZzM/edit?ts=5f479d7d#).

- Case-Study: Are there examples or best practices available how and where datasets have been successfully reused and repurposed?
- Analyse and assess - what went well, where are limitations/boundaries, to what degree can certain dataset be reused?

**Is anyone interested in expanding on this exercise and reviewing it?. Don't forget to add the solution to the lesson.**

Don't forget to check out the comments in the [google doc.](https://docs.google.com/document/d/124pu7Ufy3QKzGhowId2PtwvQ5LNJU35UpZ9qxbl1ZzM/edit?ts=5f479d7d#)

Any further comments on this, please add them to this issue!

**Comment by ischmahl** [comment]
The [StudyForrest](https://www.studyforrest.org/) dataset could be a good example for reusing data. Not only that the data is made accessible, there is also a publication list provided which includes those papers which reused the data.

**Comment by ragamouf** [comment]
Thanks @ischmahl, that's a great idea. However, I understand that this lesson is intended to be retired. Perhaps it would be better submitted to any of the stable LC lessons as a source for datasets. @jt14den @libcce what do you think?

**Comment by libcce** [comment]
Yes @ragamouf I believe it is retired. We haven't had anyone take up furthering the lesson. And some of us have moved on to other ways of communicating this information. For instance, FAIR Points https://www.fairpoints.org/ was born from discussions here.
